project/Settings.scala

```diff
@@ -438,6 +438,7 @@ object Settings {
     // We put our classes on scalac's `javabootclasspath` so that it uses them
     // when compiling rather than the definitions from the JDK.
     recompileAllOrNothingSettings,
+    scalacOptions -= "-Xfatal-warnings", // temp for ScalaOps deprecation
```

Comment:
How temporary? We should not merge the PR with this.

Reply:
It is WIP, with a plan to fix the sites and remove all the deprecated functions. Is that ok?

Reply:
Sounds like a good plan, feel free to mark the PR as a draft until it's ready for review.

Reply:
Didn't see the option. Says WIP in the title.